core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -224,7 +224,7 @@ private void exportSchema(PrintWriter out, ExportConfig config) {
             out.println(index);
         }
         if (artificialUniques > 0) {
-            String cypher = this.cypherFormat.statementForConstraint(UNIQUE_ID_LABEL, Collections.singleton(UNIQUE_ID_PROP), config.ifNotExists());
+            String cypher = this.cypherFormat.statementForConstraint(UNIQUE_ID_LABEL, Collections.singleton(UNIQUE_ID_PROP), config.ifNotExists(), StringUtils.EMPTY);
```

Comment:
Why empty constraint names here?

Reply:
Because this statement is about the `UNIQUE IMPORT LABEL`,
which is used internally in order to guarantee the uniqueness.

When the import is completed, this constraint will be removed, so retain the constraint name is unnecessary.

As said in doc:
```
When exporting nodes, 
if a node label does not contain a unique constraint the exporter will add a `UNIQUE IMPORT LABEL` label and `UNIQUE IMPORT ID` property to those nodes to ensure uniqueness of nodes when the export script is executed on a new database.
```